src/dappsExplorer/FeaturedDappCard.tsx

```diff
@@ -20,7 +20,9 @@ function FeaturedDappCard({ dapp, onPressDapp }: Props) {
       <Touchable style={styles.pressableCard} onPress={onPress} testID="FeaturedDapp">
         <>
           <View style={styles.itemTextContainer}>
-            <Text style={styles.title}>{dapp.name}</Text>
+            <Text testID="FeaturedDappCard/Name" style={styles.title}>
```

Comment:
nit: FeaturedDapp to match the testID above

Reply:
Fixed in afaa22c75524205d035e80d06bc31f3fef80a8ff!